modules/tensor_mechanics/src/actions/DomainIntegralAction.C

```diff
@@ -263,6 +266,24 @@ DomainIntegralAction::DomainIntegralAction(const InputParameters & params)
     if (!poissons_ratio_set)
       _poissons_ratio = getParam<Real>("poissons_ratio");
   }
+
+  if (_integrals.count(J_INTEGRAL) != 0 || _integrals.count(C_INTEGRAL) != 0 ||
```

Comment:
looks like enum are being used instead of enum classes

Reply:
Yes, that is the case. We should clean that up, but I'd say that should be saved for later.